### DISCUSSION TITLE
请问metrics/map50(B) 中的 （B） 是什么意思？  Please，What‘s （B）？

### DISCUSSION THREAD

**Discussion by zhuzhaohuan** [ultralytics/ultralytics#1691] [Q&A] [answered]
也没搜到这个（B）代表什么意思
I didn't find what this (B) means

**Comment by Laughing-q** [comment, marked as answer]
@zhuzhaohuan means box, map50 means the map50 of box

**Reply by zhuzhaohuan** [reply]
really thank you : D

**Reply by glenn-jocher** [reply]
@zhuzhaohuan you're welcome! 😄 If you have any more questions or need further clarification, feel free to ask. Happy detecting!
